src/templates/common/encoding/article_jats_1_2.xml

```diff
@@ -45,6 +45,17 @@
                     {% if author.is_correspondence_author %}
                     <email>{{ author.author.email }}</email>
                     {% endif %}
+                    {% for credit_record in article.creditrecord_set.all %}
+                        {% if credit_record.frozen_author == author %}
+                            <role
+                                vocab="credit"
+                                vocab-identifier="https://credit.niso.org/"
+                                vocab-term="{{ credit_record.get_role_display }}"
+                                vocab-term-identifier="{{ credit_record.uri }}">
+                                {{ credit_record.get_role_display }}
+                            </role>
+                        {% endif %}
+                    {% endfor %}
```

Comment:
This seems cumbersome, as we will repeatedly iterate all credit records and check they match the right author. can't we instead call `author.credits` and iterate that subset?.